docs/pages/toolchain/katana/interact.md

```diff
@@ -0,0 +1,201 @@
+# Interact with Katana: Deploy a simple Cairo smart contract using Katana
+
+Welcome to this tutorial where we'll guide you through deploying a raw cairo smart using katana as a local devnet. In order to accomplish this we have to install the following tools.
+
+## Install Starkli, Scarb and Katana
+
+To install `Starkli`, open a new terminal
+
+```bash
+curl https://get.starkli.sh | sh
+starkliup
+```
+
+You can check your installation by running `starkli --version`, then you will get the starkli version.
+
+To install `Scarb`
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://docs.swmansion.com/scarb/install.sh | sh
+```
+
+You can check your installation by running `scarb --version`, then you will get the starkli version.
+
+To install `Katana`, use the `dojoup` installer from the command line:
+
+```bash
```

Comment:
could you move these to a 'snippets' directory and we can reuse the snippet everywhere. https://vocs.dev/docs/guides/code-snippets

Also could you do it for other CLI commands that are generalised